src/tests/ftest/util/server_utils.py

```diff
@@ -434,7 +434,7 @@ def support_collect_log(self, **kwargs):
 
         """
         cmd = DaosServerCommand(self.manager.job.command_path)
-        cmd.sudo = False
+        cmd.sudo = True
```

Comment:
Does this require sudo now for access restrictions? Would it be okay to instead run as the `daos_server` user?

Reply:
Server is running as root via Avocado, so the server support collect-log command also needs to be run as same user to collect the daos_metrics other vice it does not collect the  daos_metrics.

Reply:
> Server is running as root via Avocado

Are you sure? It ultimately uses `sudo systemctl start daos_server`, but `systemctl` will start it as `daos_server`, not `root`.

E.g. the service file looks like
```
[Service]
Type=simple
User=daos_server
Group=daos_server
RuntimeDirectory=daos_server
RuntimeDirectoryMode=0755
ExecStart=/usr/bin/daos_server start
```

Reply:
I though the same but when I start with Avocado, service file is running as root.

`[samirrav@wolf-173 ~]$ ps -axu | grep daos
root      971593 68.0  0.1 5948556 302696 ?      Ssl  22:19   0:37 /usr/bin/daos_server start
root      974439  434  1.4 7067036 3817024 ?     SLl  22:19   2:36 /usr/bin/daos_engine -t 8 -x 4 -g daos_server -d /var/run/daos_server -T 1 -p 1 -I 1 -r 0 -H 0 -s /mnt/daos1
root      974448  403  1.4 7067036 3813352 ?     SLl  22:19   2:25 /usr/bin/daos_engine -t 8 -x 4 -g daos_server -d /var/run/daos_server -T 1 -p 0 -I 0 -r 0 -H 0 -s /mnt/daos0
samirrav  975494  0.0  0.0  12144  2328 pts/0    S+   22:20   0:00 grep --color=auto daos

[samirrav@wolf-173 ~]$ cat /usr/lib/systemd/system/daos_server.service
[Unit]
Description=DAOS Server
StartLimitIntervalSec=60
Wants=network-online.target
After=network-online.target

[Service]
Type=simple
User=root
Group=root
RuntimeDirectory=daos_server
RuntimeDirectoryMode=0755
ExecStart=/usr/bin/daos_server start
StandardOutput=journal
StandardError=journal
Restart=on-failure
RestartSec=10
LimitMEMLOCK=infinity
LimitCORE=infinity
LimitNOFILE=infinity
StartLimitBurst=5

[Install]
WantedBy = multi-user.target
`

Reply:
Is that the service file installed with the RPMs? Or where does that come from>

Reply:
It is the part of rpm. But isn't Avocado change that too run as root?

Reply:
I don't see the ftest framework changing the user, and poking through some CI wolf nodes, it looks like it's still `daos_server`. And looking at some of my nodes with RPM installs, it's `daos_server`. Are you sure that file doesn't come from somewhere else?

Reply:
You are right it's on my local system, where I change this as root to run based on source. Let me change and run that command as user daos_server.

Reply:
You should be able to change
```
cmd.sudo = True
```
to
```
cmd.run_user = "daos_server"
```

And for any "manual" command string you can do
```
from run_utils import command_as_user
command = command_as_user("do something", "daos_server")
```